README.md

```diff
@@ -67,3 +67,10 @@ Available on standard ports (eg. `5672` or `5552`) via `localhost`. The manageme
 | vhost     | /         |
 | username  | guest     |
 | password  | guest     |
+
+### PHP CodeSniffer (Linter)
+
+1. Run `./vendor/bin/phpcs /path/of/file` to evaluate the code
+2. Run `./vendor/bin/phpcbf /path/of/file` to fix code using default coding standard
+
+Note: Default coding standard is set to PSR2
```

Comment:
Have you run the linter on a sample file to make sure it works as expected?

Reply:
Yes I have. I did create a small php code. It returns all the errors and warnings, as well as fixing all those errors using phpcbf.